### PR TITLE
v4.7.0 docs link

### PR DESCRIPTION
**Pull request by thewchan**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by thewchan** [comment]
@conda-forge-admin please rerender

**Comment by thewchan** [comment]
@conda-forge/core this is ready to be merged in as the previous PRs have been sitting idle.

**Comment by ocefpaf** [comment]
> @conda-forge/core this is ready to be merged in as the previous PRs have been sitting idle.

We try to give the maintainers at least a week. Let's ping @conda-forge/pyfakefs team and give them a chance to merge this. If not please ping me again.

**Comment by thewchan** [comment]
@ocefpaf sure but just fyi the updates have been sitting idle for a few months

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,48 +1,51 @@
...
     no modification to work with pyfakefs. pyfakefs works with Linux, Windows
     and MacOS.
 
-  doc_url: http://jmcgeheeiv.github.io/pyfakefs/
-  dev_url: https://github.com/jmcgeheeiv/pyfakefs
-
 extra:
   recipe-maintainers:
     - tswast
+    - thewchan
```
Comment (tswast) [inline recipe/meta.yaml:51]: Just merged https://github.com/conda-forge/pyfakefs-feedstock/pull/29 but happy to take some of these changes, including the extra help as a maintainer.

**Review by tswast — APPROVED** [review]

**Pull request merged**